fe/fe-core/src/main/java/com/starrocks/scheduler/TaskRun.java

```diff
@@ -68,8 +69,11 @@ public class TaskRun implements Comparable<TaskRun> {
 
     private ExecuteOption executeOption;
 
+    private final String uuid;
```

Comment:
Why taskid isn't unique and we need this uuid?

Reply:
taskId is MV's id, but one task can have multi task runs which are triggered by each refresh.